src/RewardAllStakersActionGenerator.sol

```diff
@@ -70,10 +73,11 @@ contract RewardAllStakersActionGenerator is IHopperActionGenerator {
         }
 
         bEIGEN = _bEIGEN;
+        EIGEN = _EIGEN;
     }
 
-    function generateHopperActions(address hopper, address hopperToken) external view returns (HopperAction[] memory) {
-        HopperAction[] memory actions = new HopperAction[](2); 
+    function generateHopperActions(address hopper, address /*hopperToken*/) external view returns (HopperAction[] memory) {
+        HopperAction[] memory actions = new HopperAction[](5); 
 
         uint256 totalAmount;
         uint32 startTimestamp;
```

Comment:
can we have the first trigger be stateful so that the condition is `startTimestamp != 0`?

Reply:
I think you have an incorrect conception of `startTimestamp` here?